lib/plausible/stats/email_report.ex

```diff
@@ -60,4 +76,37 @@ defmodule Plausible.Stats.EmailReport do
 
     Map.put(stats, :sources, sources)
   end
+
+  defp put_top_5_goals(stats, site, query) do
+    date_str = Date.to_iso8601(query.now)
+    period_str = if query.period == "month", do: "month", else: "7d"
+
+    {:ok, q} =
+      Query.build(
+        site,
+        :internal,
+        %{
+          "site_id" => "#{site.id}",
```

Comment:
```suggestion
          "site_id" => site.domain,
```

The "site_id" we expose in the API is actually the domain (https://plausible.io/docs/stats-api#site_id-). Misnomer :sweat_smile:

Reply:
it works no matter what we pass :shrug:

Reply:
https://github.com/plausible/analytics/pull/5696/commits/5b7eb0165c1748529c0a505f0925c32257f065dc